src/components/pages/AddFarmer/AddFarmer.js

```diff
@@ -48,6 +48,18 @@ const AddFarmer = () => {
       } else {
         newEntry.selected = [...newEntry.selected, value];
       }
+    } else if (type === 'file') {
```

Comment:
This could be problematic if we add other files in the futrue, but for MVP its no problem, still worth to keep in mind

Reply:
Ok